bgs/bgs.go

```diff
@@ -347,7 +342,7 @@ func (bgs *BGS) StartWithListener(listen net.Listener) error {
 				sendHeader = false
 			}
 
-			bgs.log.Warn("HANDLER ERROR: (%s) %s", ctx.Path(), err)
+			bgs.log.Warn("HANDLER ERROR", "path", ctx.Path(), "err", err)
```

Comment:
Structured (not formatted-string) logging, here and in a couple of other spots below too.